Library/Formula/websphere-liberty-profile.rb

```diff
@@ -0,0 +1,70 @@
+class WebsphereLibertyProfile < Formula
+  desc "WebSphere Liberty Profile"
+  homepage "https://developer.ibm.com/wasdev/websphere-liberty/"
+  url "https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/8.5.5.7/wlp-javaee7-8.5.5.7.zip"
+  sha256 "77f710c93cfd45661782d9895d5a17caa751a4186003753044e06b730d380bba"
+
+  depends_on :java => "1.7+"
+
+  def install
+    rm_rf Dir["bin/*.bat"]
+    rm_rf "usr"
+    rmdir "usr"
+
+    binaries = %w[binaryLog collective ddlGen installUtility securityUtility serverSchemaGen client configUtility featureManager isadc productInfo server]
+    # Replace to specify the install path directly
+    #
+    # WLP_INSTALL_DIR=`unset CDPATH; cd "$dirname/../" && pwd`
+    # or
+    # WLP_INSTALL_DIR=`unset CDPATH; cd "$dirname/.." && pwd`
+
+    binaries.each do |name|
+      inreplace "bin/#{name}" do |s|
+        s.gsub! %r{WLP_INSTALL_DIR=`unset CDPATH; cd \"\$dirname/\.\./?\" && pwd`}, "WLP_INSTALL_DIR=#{libexec}"
+      end
+    end
+
+    libexec.install Dir["*"]
+
+    # do not use bin.install because IDE integration may require the path to the scripts be in the same place.
+    binaries.each do |name|
+      bin.install_symlink "#{libexec}/bin/#{name}"
```

Comment:
So just change the name when we install right?  
May I use `wlp-server` and `wlp-client` it's not WebSphere Liberty but WebSphere Liberty Profile.